### DISCUSSION TITLE
How long does it usually take for a PR to get re-reviewed?

### DISCUSSION THREAD

**Discussion by ron0studios** [TheAlgorithms/C-Plus-Plus#2275] [Q&A]
There's are a **lot** of PRs coming into this repository, with a couple having made the relevant PR changes after the initial review. How long does it usually take for said pull requests to get re-reviewed? 
Thanks,

**Comment by Panquesito7** [comment]
We're sorry for the delay in reviewing PRs. We currently don't have many active maintainers.
I'm trying to get through all the PRs (a bit late, though). You can help us with the process if you'd like to. 🙂

Thanks. Have a nice day.

**Comment by Veercodeprog** [comment]
The time it takes for a pull request to get re-reviewed can vary depending on several factors, such as the size and complexity of the changes, the responsiveness of the project maintainers, and the overall workload of the project maintainers.
In some cases, a pull request may be re-reviewed and merged quickly, within a few hours or days. In other cases, it may take several weeks or even months for a pull request to be re-reviewed and merged.
